tests/repository_data/uptane_director_root.json

```diff
@@ -0,0 +1,86 @@
+{
```

Comment:
Root metadata is now always written as if if consistent snapshots = True.  Therefore, expect to have `1.uptane_director_root.json`, `2.uptane_director_root.json` for this repository.